R/addins.R

```diff
@@ -1,8 +1,112 @@
-read_addin <- function(...) {
-  path <- fs::path_package("rhino", "rstudio", "addins", ...)
-  readChar(path, file.info(path)$size)
+run_background <- function(file) {
+  path <- fs::path_package("rhino", "rstudio", "addins", file)
+  rstudioapi::jobRunScript(path, workingDir = rstudioapi::getActiveProject())
 }
 
 addin_module <- function() {
-  rstudioapi::insertText(read_addin("module.R"))
+  module_path <- fs::path_package("rhino", "rstudio", "addins", "module.R")
+  file_path <- rstudioapi::selectFile(
```

Comment:
The addin needs to handle the case when a user cancels the file selection. In this case `selectFile` will return `NULL`.

Reply:
Done!